### ISSUE TITLE
Nested Checklists and Lists don't Align with non-nested ones

### ISSUE THREAD

**Issue by s-kyy** [radekkozak/discordian#42]
OS: Windows
Discordian version: 0.8.5
Obsidian: Version: 0.11.13

When I want to see checklists and unordered lists in preview, the nested checklist icons and text is more indented than the non-nested checklist items. For ordered and unordered lists, the icon is aligned while the text is not. 

Here is an example of text in the markdown:
```
- [ ] task 3
	- [ ] task 2
- [ ] task 2
- [ ] task 1
	- [ ] task 2
	- [ ] task 3

New List
- line 1
	- line 1.1
- line 2
- line 3

1. Item 1
	1. Item 1.1
2. Item 2
```

Here's what the Preview mode looks like:
![image](https://user-images.githubusercontent.com/19920083/113951553-c2688e80-97e1-11eb-84e4-be123a566221.png)

**Comment by radekkozak** [comment]
Hi @s-kyy
very sorry for long delay on this one (explanation in other issues) 

I just managed to work out the problem you described. The quick fix is coming up with 0.8.6 release today. I will keep you posted. Oh, and thank you very much for keeping your finger on this one and contributing. Very much helpful for keeping the theme up to date and working as expected

All best from pandemic Poland

**Comment by s-kyy** [comment]
Thanks @radekkozak, looking forward to the update! Glad to hear that you're recovering, but definitely take it easy if you don't feel well. 

Best wishes
